src/nerian_stereo_node.cpp

```diff
@@ -769,8 +777,8 @@ void StereoNode::publishCameraInfo(rclcpp::Time stamp, const ImageSet& imageSet)
         }
     }
 
-    double dt = (stamp - lastCamInfoPublish).seconds();
-    if(dt > 1.0) {
+    double dt = (stamp.seconds() - lastCamInfoPublish.seconds());
+    if(dt > 0.50){
```

Comment:
Could this value be passed in a config file?

Reply:
This data does not need to be changed frequently